### PR TITLE
21 react router

### PR DESCRIPTION
**Pull request by cameronbeattie306**
I followed this video for the react-router: https://www.youtube.com/watch?v=G7UzhrNX60o&ab_channel=TeddySmith
(this video also goes over how to implement the navbar)

For some reason, the react-router implementation I used for 370 didn't work so any tips would be greatly appreciated

Thanks guys!

### CODE REVIEW

File: frontend/src/Routes/Routes.tsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { createBrowserRouter } from "react-router-dom";
+import App from "../App";
+import TestPage from "../pages/TestPage";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <App />,
+    children: [{ path: "testPage", element: <TestPage /> }],
```
Comment (ralphgregorio) [inline frontend/src/Routes/Routes.tsx:10]: Looks correct but I'm trying to test the path /testPage and its not working. What is the correct link to access this?

**Review by ralphgregorio — COMMENTED** [review]
For some reason i am not able to change links. Also can you change this PR to merge into development branch? so it hides my changes and it merges to the correct branch

**Review by ralphgregorio — APPROVED** [review]
looks gucci

**Review by glenntanjoh — APPROVED** [review]
Looks good, well placed. Thanks for the support!

**Pull request merged**